src/AutoBuild.jl

```diff
@@ -1268,6 +1278,10 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
         print(io, jll_jl)
     end
 
+    print_source(io, s::ArchiveSource) = println(io, "* compressed archive ", s.url, " (SHA256 checksum: `", s.hash,"`)")
+    print_source(io, s::GitSource) =     println(io, "* git repository ", s.url, " (revision: `", s.hash,"`)")
+    print_source(io, s::FileSource) =    println(io, "* file ", s.url, " (SHA256 checksum: `", s.hash,"`)")
+    print_source(io, ::DirectorySource) = nothing
```

Comment:
I was not sure whether we want to list also the list of local directories, as we don't have a direct reference from here.  I'm open to suggestions

Reply:
It would be nice if we list the files inside local directories, but I don't know of that's possible.

Reply:
Not really, not now at least: we don't have the list of files in the directory anywhere.

Reply:
Let's list the name of the directory, and if we're building from Yggdrasil, we can auto-detect the gitsha of the current commit, then emit a link to the directory at that commit.